src/content/reference/developer-tooling/react-performance-tracks.md

```diff
@@ -0,0 +1,137 @@
+---
+title: React Performance tracks
+version: experimental
+---
+
+<Experimental>
+
+**This feature is experimental and is not available in a stable version of React yet.**
+
+You can try it by upgrading React packages to the most recent experimental version:
+
+- `react@experimental`
+- `react-dom@experimental`
+- `eslint-plugin-react-hooks@experimental`
+
+Experimental versions of React may contain bugs. Don't use them in production.
+
+</Experimental>
+
+<Intro>
+
+React Performance tracks are specialized custom entries that appear on the Performance panel's timeline.
+
+</Intro>
+
+These tracks are designed to provide developers with comprehensive insights into their React application's performance by visualizing React-specific events and metrics alongside other critical data sources such as network requests, JavaScript execution, and event loop activity, all synchronized on a unified timeline within the Performance panel for a complete understanding of application behavior.
+
+<div style={{display: 'flex', justifyContent: 'center', marginBottom: '1rem'}}>
+  <img className="w-full light-image" src="/images/docs/performance-tracks/overview.png" alt="React Performance Tracks" />
+  <img className="w-full dark-image" src="/images/docs/performance-tracks/overview.dark.png" alt="React Performance Tracks" />
+</div>
+
+<InlineToc />
+
+---
+
+## Usage {/*usage*/}
+
+React Performance tracks are only available in development and profiling builds of React:
+
+- **Development**: enabled by default.
+- **Profiling**: you can either wrap a subtree that you want to instrument with [`<Profiler>`](/reference/react/Profiler) component or have [React Developer Tools extension](/learn/react-developer-tools) enabled.
+
+If enabled, tracks should appear automatically in the traces you record with the Performance panel of browsers that provide [extensibility APIs](https://developer.chrome.com/docs/devtools/performance/extension).
+
+<Pitfall>
+
+The profiling instrumentation that powers React Performance tracks adds some additional overhead, so it is disabled in production builds by default.
+
+</Pitfall>
+
+---
+
+## Tracks {/*tracks*/}
+
+### Scheduler {/*scheduler*/}
+
+The Scheduler is an internal React concept used for managing tasks with different priorities. This track consists of 4 subtracks, each representing work of a specific priority:
+
+- **Blocking** - The synchronous updates, which could've been initiated by user interactions.
```

Comment:
State updates from commits (layout effects, ref assignments, etc.) are also scheduled with a blocking priority. We should probably mention that too.

Reply:
The "The synchronous updates" are referring to the application updates, not to the state updates.

I am not sure how to update it to clarify it more